faker/providers/credit_score/__init__.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import BaseProvider
+
+
+class CreditScore(object):
```

Comment:
This could be made more memory-efficient by using a `namedtuple`

Reply:
Which part(s) would you like to see refactored as a namedtuple? Want to make sure I'm not overthinking this.

Reply:
instead of:
```python
class CreditScore(object):
    # ...
```

declare `CreditScore` by using `namedtuple`:

```python
CreditScore = namedtuple('CreditScore', ['name', 'provider', 'score_range'])
```

Everything else should just work :)

Reply:
Got it -- I thought you meant the OrderedDict and was scratching my head over how to refactor that in a way that made it cleaner as a namedtuple.